sdk/storage/Azure.Storage.Common/api/Azure.Storage.Common.netstandard2.0.cs

```diff
@@ -101,8 +102,10 @@ public enum SasProtocol
     }
     public partial class SasQueryParameters
     {
-        internal SasQueryParameters() { }
         public const string DefaultSasVersion = "2019-02-02";
+        protected SasQueryParameters() { }
+        public SasQueryParameters(System.Collections.Generic.Dictionary<string, string> values) { }
+        public SasQueryParameters(string version, Azure.Storage.Sas.AccountSasServices? services, Azure.Storage.Sas.AccountSasResourceTypes? resourceTypes, Azure.Storage.Sas.SasProtocol protocol, System.DateTimeOffset startsOn, System.DateTimeOffset expiresOn, Azure.Storage.Sas.SasIPRange ipRange, string identifier, string resource, string permissions, string signature, string cacheControl = null, string contentDisposition = null, string contentEncoding = null, string contentLanguage = null, string contentType = null) { }
```

Comment:
Let's make both of these .ctors protected if we can and then use `protected static SasQueryParameters Create(...)` for the builders to call them.

Reply:
Updated